exonum/src/node/state.rs

```diff
@@ -851,6 +909,35 @@ impl State {
         }
     }
 
+    /// Finds unknown transactions in the block.
+    pub fn create_incomplete_block(
```

Comment:
Since this function can panic, please add a note about it to the docs (like it's done in `add_prevote`). 
BTW, should it really panic? Maybe `error!(...)` will be enough here?

Reply:
Sorry for the confusion :sweat:  If I don't miss anything, the situation can occur only if a qualified majority of validators decided to include some transaction in the blockchain twice. This is a reason for panic, imo; we don't have a valid way to arrive to this point and don't know how to deal with such a transaction.